src/domains/transaction/components/TransactionSuccessful/TransactionSuccessful.test.tsx

```diff
@@ -20,7 +19,12 @@ describe("TransactionSuccessful", () => {
 		await env.profiles().restore(profile);
 		await profile.sync();
 
-		vi.spyOn(useConfirmedTransactionMock, "useConfirmedTransaction").mockReturnValue(true);
+		server.use(
+			requestMock(
+				"https://ark-test.arkvault.io/api/transactions/ea63bf9a4b3eaf75a1dfff721967c45dce64eb7facf1aef29461868681b5c79b",
+				transactionsFixture,
+			),
+		);
```

Comment:
It is not responsibility of arkvault to ensure PSDK functions properly, it is a unit  test and should be limited to the SUT, it will introduce more breaking changes when we change stuff in PSDK, at the end of the day PSDK is a library and it has own tests, it is not up to consumer to ensure its behavior.. 
Also mocking it is not muting problem it is called isolating side factors, you have to rule out some factors when doing unit tests otherwise changing a single function implementation would cause hundreds of tests fail